stable/ambassador/values.yaml

```diff
@@ -1,3 +1,4 @@
+---
```

Comment:
why this? 🤔

Reply:
to make my YAML linter happy!  i can absolutely remove it if you prefer

Reply:
please do, just to avoid confusion 😄